src/constants.ts

```diff
@@ -73,6 +73,7 @@ export const STABLE_COIN_SYMBOLS = [
   TOKEN_SYMBOLS_MAP.USDB.symbol,
   TOKEN_SYMBOLS_MAP.USDC.symbol,
   TOKEN_SYMBOLS_MAP.USDT.symbol,
+  TOKEN_SYMBOLS_MAP["USDT-BNB"].symbol,
```

Comment:
@nicholaspai Do we also need USDC-BNB in here?

Reply:
I added it in a previous commit here https://github.com/across-protocol/sdk/blob/7457901f003e714bfaeae0614386cea85900a27e/src/constants.ts#L68, but I forgot to add USDT.